packages/icon/src/vaadin-icon.js

```diff
@@ -110,7 +110,10 @@ class Icon extends ThemableMixin(ElementMixin(ControllerMixin(SlotStylesMixin(Ic
         viewBox="[[__computeViewBox(size, __viewBox)]]"
         preserveAspectRatio="[[__computePAR(__defaultPAR, __preserveAspectRatio)]]"
         aria-hidden="true"
-      ></svg>
+      >
+        <g id="svg-group"></g>
+        <g id="use-group"></g>
```

Comment:
Decided to add two groups inside the SVG element so having the sprite SVG along with the `<use>` element can be done by simply rendering the parts in the different groups.